Mars Mission/Mars Mission/APILayer/APILayer.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  APILayer.swift
+//  Mars Mission
+//
+//  Created by Aldrin Bahle Gama on 2020/11/04.
+//
+
+import Foundation
+
+struct API {
+    static let nasaAPI = "NYVu6FVU9WwiBBy9zp4iaWveE1SmWlnDFgEy32cH"
```

Comment:
Consider putting these URL end points within a plist

Reply:
I've resolved this one and the screenshots of the changes are at the top.